_el/exercises.md

```diff
@@ -336,172 +204,52 @@ piece of code with the output it produces".
+
+Τα προβλήματα αντιστοίχισης μπορούν να υλοποιηθούν με την υποβολή από μαθητές καταλόγων με ζευγάρια που εκφράζουν τις αντιστοιχίσεις ως κείμενο (όπως για παράδειγμα "A3, B1, C2"), αλλά αυτός είναι ένας γενικά αδέξιος τρόπος που είναι επιρρεπής σε λάθη. Ζητώντας να αναγνωρίσουν ένα σύνολο σωστών ζευγών υπό τη μορφή ΕΠΕ είναι ακόμα χειρότερο, καθώς είναι δυστυχώς πολύ εύκολο να το διαβάζει κανείς εσφαλμένα.
+
+Η *Κατάταξη* είναι μια ειδική περίπτωση που (ελαφρώς) ταιριάζει σε απάντηση μέσω λιστών, καθώς το μυαλό μας είναι πολύ καλά εκπαιδευμένο στο να εντοπίζει σφάλματα ή ανωμαλίες σε ακολουθίες. Δώστε στο μαθητή πολλά στοιχεία και ζητήσετε να τα κατατάξουν από τα ταχύτερα στα πιο αργά, από τα πιο ανθεκτικά στα πιο εύθραυστα, και ούτω καθεξής. Η πρώτη μορφή ενδείκνυται περισσότερο για την ανάκληση πληροφορίας (π.χ. την αναγνώριση των ονομάτων διαφόρων αλγορίθμων κατάταξης και γνωρίζοντας τις ιδιότητές τους), ενώ η δεύτερη τείνει περισσότερο προς τη συλλογιστική και την κρίση.
+
+Η *Σύνοψη* απαιτεί επίσης από τους εκπαιδευόμενους να χρησιμοποιούν συλλογιστική υψηλότερης τάξης, και τους δίνει την ευκαιρία να εξασκήσουν μια δεξιότητα που είναι πολύ χρήσιμη όταν στοχεύει περισσότερη στην ανίχνευση σφαλμάτων παρά στη διόρθωσή τους. Για παράδειγμα, η ερώτηση προς τους μαθητές μπορεί να είναι: "Ποια πρόταση περιγράφει καλύτερα τον τρόπο με τον οποίο η έξοδος της f αλλάζει όταν το x παίρνει τιμές μεταξύ 0 και 10;" και δίνοντάς τους μετά πολλές επιλογές ως ερώτηση πολλαπλών επιλογών.
+
+Μπορείτε επίσης να ζητήσετε πολύ σύντομες απαντήσεις ελεύθερου κειμένου σε ερωτήσεις που έχουν πολύ στενό αντικείμενο, όπως για παράδειγμα "Ποιο είναι το βασικό χαρακτηριστικό ενός ευσταθούς αλγορίθμου ταξινόμησης;" Δεν μπορούμε ακόμα να αυτοματοποιήσουμε πλήρως τη βαθμολόγηση τέτοιων ερωτήσεων χωρίς ένα
+απογοητευτικό αριθμό ψευδών θετικών (αποδοχή λανθασμένων απαντήσεων) και ψευδών αρνητικών (απορρίπτοντας τα σωστά), αλλά ενδείκνυνται για αυτοαξιολόγηση ([s:individual-peer](#SECTION)).
+
+## Αυτόματη βαθμολόγηση {#s:exercises-grading}
+
+Τα αυτόματα εργαλεία βαθμολόγησης του προγράμματος υπάρχουν πολύ πριν γεννγθώ εγώ: η παλαιότερη δημοσιευμένη αναφορά χρονολογείται από το 1960 [[Holl1960](#CITE)], και οι έρευνες που δημοσιεύθηκαν στα [[Douc2005](#CITE),[Ihan2010](#CITE)] αναφέρουν ονομαστικά πολλά εργαλεία με αυτό το στόχο. Η κατασκευή τέτοιων εργαλείων είναι πολύ πιο πολύπλοκη από όσο μπορεί να φαίνεται αρχικά. Πως μπορούν να αντιπροσωπεύονται οι αναθέσεις; Πώς γίνεται η καταγραφή και παρακολούθησετι των απαντήσεων που υποβάλλονται; Μπορούν οι μαθητές να συνεργαστούν; Πώς μπορούν να γίνουν οι υποβολές απαντήσεων με ασφάλεια; [[Edwa2014a](#CITE)] είναι ένα άρθρο που αφιερώνεται εξ' ολοκλήρου σε ένα προσαρμοστικό σχήμα για την ανίχνευση και διαχείριση ατέρμονων βρόχων και άλλων μορφών κώδικα που δεν τερματίζουν κανονικά, και αυτό είναι μόνο ένα από τα πολλά ζητήματα που αναφέρονται.
```

Comment:
γεννγθώ-->γεννηθώ
παρακολούθησετι--> παρακολούθηση
[[Edwa2014a](#CITE)]--> Η εργασία [[Edwa2014a](#CITE)]

Reply:
Τhanks @pdelias ! Fixed in a new commit.